docs/getting-started.md

```diff
@@ -33,6 +40,34 @@ If you are using Git to manage your Hugo site, add this theme as a submodule.
 git submodule add https://github.com/CaiJimmy/hugo-theme-stack/ themes/hugo-theme-stack
 ```
 
+## Route 2 (recommended): use theme as Hugo module
+
+- Turn your new site into a Hugo module:
+```bash
+hugo mod init github.com/me/my-new-blog
+```
+
+- Declare the `hugo-theme-stack` module as a dependency of your site:
+```bash
+hugo mod get github.com/CaiJimmy/hugo-theme-stack
+```
+
+- Add the following lines at the end of your `config.yaml`:
+
+```yaml
+module:
+  # uncomment line below for temporary local development of module
+  # replacements: "github.com/CaiJimmy/hugo-theme-stack -> ../../hugo-theme-stack"
+  hugoVersion:
+    extended: true
+    min: 0.78.0
```

Comment:
I think this block (`hugoVersion`) isn't necessary for the users to include it. It should be the theme to decide which is the minimum required Hugo version.

Reply:
> I think this block (`hugoVersion`) isn't necessary for the users to include it.

You are right. I removed this block from the config file.